dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/CpuInfo.java

```diff
@@ -39,14 +36,12 @@
      */
     private final static TraceComponent tc = Tr.register(CpuInfo.class);
 
-    private static CpuInfo instance;
-    private final int AVAILABLE_PROCESSORS;
+    private static CpuInfo INSTANCE = new CpuInfo();
```

Comment:
Should this be marked final?

Reply:
It probably should be, although it wasn't before and it probably should have been. If I have to make changes for IBM Java 7 support (your other comment) then I'll address it for sure.

Reply:
It was not final before because it was lazily initialized instead of eagerly initialized like you are doing in your changes.